.github/workflows/tests.yml

```diff
@@ -90,6 +92,11 @@ jobs:
             if (label_names.includes('CI-published-bottle-commits')) {
               console.log('CI-published-bottle-commits label found. Skipping tests job.')
               core.setOutput('syntax-only', 'true')
+              syntax_only = true
+            }
+
+            if (syntax_only) {
+              return
             }
```

Comment:
With this early return, subsequent `setOutput` calls are skipped, leaving some of the output variables of this step undefined (?). But I tried a simplified workflow run in https://github.com/ZhongRuoyu/actions-test/actions/runs/4521005463/jobs/7962485880. That seems to be fine.

Reply:
@ZhongRuoyu I dunno, that feels pretty bad? What about just skipping the actual e.g. long build handling logic instead?

Reply:
I've thought about that, but I was afraid of complicating the long build handling logic. (Currently this script is divided into blocks, each handling its corresponding label / set of labels.)

I chose to do this instead because I didn't see the necessity to check for other labels when `syntax-only` is true. So I was effectively making an implicit assumption that the `syntax-only` output would be checked before anything else. The workflow is valid as long as this assumption still holds. What do you think, @MikeMcQuaid?

Reply:
@ZhongRuoyu If there's no other visible consequences: happy to leave this as-is until it's a problem, thanks 👍🏻